integration-cli/docker_cli_inspect_test.go

```diff
@@ -9,36 +9,38 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/docker/docker/runconfig"
 	"github.com/go-check/check"
 )
 
+func checkValidGraphDriver(c *check.C, name string) {
+	if name != "devicemapper" && name != "overlay" && name != "vfs" && name != "zfs" && name != "btrfs" && name != "aufs" {
+		c.Fatalf("%v is not a valid graph driver name", name)
+	}
+}
+
 func (s *DockerSuite) TestInspectImage(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	imageTest := "emptyfs"
 	imageTestID := "511136ea3c5a64f264b78b5433614aec563103b4d4702f3ba7d4d2698e22c158"
 	id, err := inspectField(imageTest, "Id")
-	c.Assert(err, check.IsNil)
+	c.Assert(err, checker.IsNil)
 
-	if id != imageTestID {
-		c.Fatalf("Expected id: %s for image: %s but received id: %s", imageTestID, imageTest, id)
-	}
+	c.Assert(id, checker.Equals, imageTestID)
 }
 
 func (s *DockerSuite) TestInspectInt64(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	out, _, err := dockerCmdWithError("run", "-d", "-m=300M", "busybox", "true")
```

Comment:
This should just use `dockerCmd` instead of `dockerCmdWithError`

Also for cases like this, there is no need to do that much with the comment since the output when it is a failure is mostly sufficient... generally I do `c.Assert(err, check.IsNil, check.Commentf(out))`